extensions/default/src/DicomWebDataSource/wado/retrieveMetadataLoader.js

```diff
@@ -37,13 +43,9 @@ export default class RetrieveMetadataLoader {
   async runLoaders(loaders) {
     let result;
     for (const loader of loaders) {
-      try {
-        result = await loader();
-        if (result && result.length) {
-          break; // closes iterator in case data is retrieved successfully
-        }
-      } catch (e) {
-        throw e;
+      result = await loader();
```

Comment:
why we are getting rid of the try/catch

Reply:
Linting was annoying me that it was useless